tests/integration/test.changes.js

```diff
@@ -1358,77 +1357,71 @@ adapters.forEach(function (adapter) {
               "changes": [{ "rev": "3-f"}
               ]
             }
-          ]
-          //, "last_seq": seqs[5]
+          ],
+          "last_seq": seqs[5]
```

Comment:
ahhhh this explains it! I could not figure out what was wrong with the descending changes. nice one.